.github/workflows/ci.yml

```diff
@@ -110,6 +112,23 @@ jobs:
         run: yarn lerna run test-unit-ibmi --scope=@sequelize/core
       - name: Unit tests (snowflake)
         run: yarn lerna run test-unit-snowflake --scope=@sequelize/core
+  unit-test-win:
+    strategy:
+      fail-fast: false
+      matrix:
+        node-version: [16, 20]
+    name: Simple Windows Unit test all dialects (Node ${{ matrix.node-version }})
+    runs-on: windows-latest
+    steps:
+      - uses: actions/checkout@c85c95e3d7251135ab7dc9ce3241c5835cc595a9 # v3.5.3
+      - uses: actions/setup-node@e33196f7422957bea03ed53f6fbb155025ffc7b8 # v3.7.0
+        with:
+          node-version: ${{ matrix.node-version }}
+          cache: yarn
+      - name: Install dependencies
+        run: yarn install --immutable
+      - name: Lerna Unit Tests
+        run: yarn lerna run test-unit
```

Comment:
Maybe only unit test sqlite? That reduces some time as well. You'll have to run a few commands for it though

Reply:
You commented on this twice, once saying `as well` and the second time saying `only`...
I've currently updated it to do the whole `test-unit` + `test-unit-sqlite --scope=@sequelize/core`.
It's 1min 30sec for the first and 12sec for the second.

Reply:
One comment was to replace running unit tests on all dialects with running unit tests only for sqlite. The other comment was about integration tests

Reply:
They were both on line 128-131.

Reply:
Ohh you mean I should add an OS matrix to `test-sqlite`?

Reply:
No, I meant that you can replace the commands after the `yarn install` with;
```yml
- name: ESM / CJS export equivalence
  run: yarn test-unit-esm
- name: Unit tests (validator.js)
  run: yarn lerna run test-unit --scope=@sequelize/validator.js
- name: Unit tests (sqlite)
  run: yarn lerna run test-unit-sqlite --scope=@sequelize/core
- name: Integration tests (sqlite)
  run: yarn lerna run test-integration-sqlite --scope=@sequelize/core
```

Reply:
Done in commit [75a783](https://github.com/sequelize/sequelize/pull/16283/commits/75a783b937c4800f47533512217ea85d5917b4b8)